lang/sbcl/Portfile

```diff
@@ -35,17 +35,24 @@ homepage        http://www.sbcl.org
 patchfiles      0001-fix-building-when-root-directory-contain-non-ASCII-c.patch \
                 0002-add-MacPorts-XDG_DATA_DIRS.patch
 
+# DEPRECATED: stopped working in sbcl-2.5.9; upstream pull request was
+#             closed. Assuming that this patch is no longer relevant,
```

Comment:
@easye AFAICT, it is not closed, it is just ignored: https://github.com/sbcl/sbcl/pull/62

I will check if that is still relevant.

Reply:
please remove the patchfile and code here instead of commenting it out.

Reply:
Will do when I have tested @barracuda156 changes as well.

Reply:
Removed.